link/tracepoint_test.go

```diff
@@ -66,7 +66,7 @@ func TestTracepointErrors(t *testing.T) {
 }
 
 func TestTraceGetEventID(t *testing.T) {
-	_, err := getTraceEventID("syscalls", "sys_enter_open")
+	_, err := getTraceEventID("syscalls", "sys_enter_openat")
```

Comment:
Why?

Reply:
Same as above, just a more widely implemented syscall, that is available on other archs.